jbi/models.py

```diff
@@ -41,6 +48,13 @@ class ActionSteps(BaseModel):
         "create_comment",
     ]
 
+    @validator("*")
+    def validate_steps(cls, function_names):
+        for function_name in function_names:
+            if not steps_module_has_function(function_name):
+                raise ValueError(f"{function_name} is not a valid step function")
```

Comment:
nitpick: raise a valuerror with all invalid step names (in order to avoid several loops of config change / fix / redeploy)